app/controllers/hq/sponsors_controller.rb

```diff
@@ -47,6 +48,12 @@ def load_sponsor
     @sponsor= Sponsor.find(params[:id])
   end
 
+  def load_active_and_inactive_sponsorships
+    sponsorships = Sponsorship.where("sponsor_id = ?", @sponsor.id).eager_load(:orphan, :sponsor).all
```

Comment:
or we can use:
`sponsorships = Sponsorship.joins(:sponsor, :orphan).where("sponsor_id = ?",@sponsor.id).select("sponsorships.*, orphans.name as orphans_name, orphans.date_of_birth as orphans_date_of_birth, orphans.gender as orphans_gender")`